staking/programs/integrity-pool/src/state/pool.rs

```diff
@@ -375,6 +375,31 @@ impl PoolData {
         Ok(())
     }
 
+    pub fn apply_slash(
+        &mut self,
+        publisher: &Pubkey,
+        stake_account_positions_key: &Pubkey,
+        locked_slash: u64,
```

Comment:
nit: this is called `locked_slashed` in the staking program and `locked_slash` here

Reply:
fixed